packages/stats/NAMESPACE

```diff
@@ -25,7 +25,19 @@ useDynLib(stats, .registration = FALSE, .fixes="C_",
     SplineEval,
     monoFC_m,
     cov,
-    cor
+    cor,
+    KalmanLike,
+    KalmanSmooth,
+    KalmanFore,
+    ARIMA_undoPars,
+    ARIMA_transPars,
+    ARIMA_Invtrans,
+    ARIMA_Gradtrans,
+    ARIMA_Like,
+    ARIMA_CSS,
+    TSconv,
+    getQ0,
```

Comment:
Are all of these functions actually exported and used in the R code, or are some of them internal to the C code? Check the stat's package NAMESPACE file in GNU R:
https://github.com/wch/r-source/blob/trunk/src/library/stats/NAMESPACE